python/ray/data/_internal/stats.py

```diff
@@ -694,7 +694,7 @@ def to_summary(self) -> "DatasetStatsSummary":
 @DeveloperAPI
 @dataclass
 class DatasetStatsSummary:
-    stages_stats: List["StageStatsSummary"]
+    operator_stats: List["OperatorStatsSummary"]
```

Comment:
nit: `operators_stats` to retain plurality, since this is holding stats for multiple operators.